mtk_logger.cc

```diff
@@ -617,7 +617,7 @@ static void mtk_read()
     dsize = 0;
     init_scan = 0;
     rename(TEMP_DATA_BIN, TEMP_DATA_BIN_OLD);
-    dout = fopen(TEMP_DATA_BIN, "wb");
```

Comment:
should we use QFile::rename here like we do in other formats?  I suspect rename may not work on WIN32 with utf8 names.